packages/prompts/test/autocomplete.test.ts

```diff
@@ -0,0 +1,124 @@
+import { afterEach, beforeEach, describe, expect, test, vi } from 'vitest';
+import { autocomplete } from '../src/autocomplete.js';
+import { MockReadable, MockWritable } from './test-utils.js';
+
+describe('autocomplete', () => {
+	let input: MockReadable;
+	let output: MockWritable;
+	const testOptions = [
+		{ value: 'apple', label: 'Apple' },
+		{ value: 'banana', label: 'Banana' },
+		{ value: 'cherry', label: 'Cherry' },
+		{ value: 'grape', label: 'Grape' },
+		{ value: 'orange', label: 'Orange' },
+	];
+
+	beforeEach(() => {
+		input = new MockReadable();
+		output = new MockWritable();
+	});
+
+	afterEach(() => {
+		vi.restoreAllMocks();
+	});
+
+	test('renders initial UI with message and instructions', async () => {
+		const result = autocomplete({
+			message: 'Select a fruit',
+			options: testOptions,
+			input,
+			output,
+		});
+
+		expect(output.buffer).toMatchSnapshot();
+	});
+
+	test('shows placeholder when provided', async () => {
```

Comment:
i don't think we actually render the placeholder

the snapshot shows this

so we should decide either:
- remove this test and autocomplete doesn't support placeholders
- make placeholders work and update this test's snapshot

Reply:
i opted for removing the test 🫡